src/App.hs

```diff
@@ -69,7 +69,7 @@ instance ToJSONKey Responder
 data State = State { lectureName     :: String
                    , timeSlots       :: [TimeRange]
                    , conferenceUrl   :: String
-                   , activeRequests  :: Object Responder HelpRequest
+                   , activeRequests  :: HM.HashMap Responder HelpRequest
```

Comment:
This is probably a problem. [This](https://hackage.haskell.org/package/aeson-1.4.6.0/docs/Data-Aeson.html#t:Object), seems to imply that the map should be mapping `Text` to `Value`.

Also, "HashMap" seems to be deprecated.